spyder/utils/ipython/tests/test_spyder_kernel.py

```diff
@@ -9,13 +9,35 @@
 """
 
 import os
-
+import os.path as osp
 import pytest
+import tempfile
 
 from spyder.config.main import CONF
 from spyder.py3compat import PY2, is_binary_string
 from spyder.utils.encoding import to_fs_from_unicode
 from spyder.utils.ipython.kernelspec import SpyderKernelSpec
+from spyder.utils.misc import get_python_executable
+
+# Temporary directory
+TEMPLOCATION = tempfile.gettempdir()
+INTERPRETER = osp.join(TEMPLOCATION, "interpreter")
```

Comment:
You can use the `tmpdir` fixture instead of this, like I did here:

https://github.com/spyder-ide/spyder/blob/master/spyder/app/tests/test_mainwindow.py#L216